cosinnus_message/rocket_chat.py

```diff
@@ -100,7 +135,15 @@ class RocketChatConnection:
 
     rocket = None
     stdout, stderr = None, None
-    
+
+    # Reusable user message when rocketchat is considered down
+    ROCKET_CHAT_DOWN_USER_MESSAGE = _('RocketChat integration is currently not working. Please try again later.')
```

Comment:
I would suggest using "We are currently experiencing some technical difficulties with the RocketChat service and some of your actions may not have had an effect. Please try again later. We aplogize for the inconveniences!" as a more detailed error message. What do you think?